third_party/absl/CMakeLists.txt

```diff
@@ -11,6 +11,11 @@ list(APPEND CMAKE_ARGS
     "-DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}"
     "-DCMAKE_CXX_STANDARD=17"
     "-DABSL_PROPAGATE_CXX_STD=ON"
+    "-DABSL_BUILD_TESTING=OFF"
+    "-DBUILD_TESTING=OFF"
+    "-DABSL_BUILD_TEST_HELPERS=ON"
+    "-DABSL_USE_EXTERNAL_GOOGLETEST=ON"
+    "-DABSL_FIND_GOOGLETEST=ON"
```

Comment:
Hmm shouldn't that be set to OFF, if BUILD_TESTING is set to OFF? It seems like the CI is failing because gtest is not found 🤔. Otherwise you need to move [gtest](https://github.com/mavlink/MAVSDK/blob/main/third_party/CMakeLists.txt#L32) up the list, but it feels unnecessary here.

Reply:
Yes, I agree. And I tried that.

Reply:
Good hint on moving it up though. I'll try that.